app/csv.go

```diff
@@ -0,0 +1,25 @@
+package app
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+)
+
+func CSVReader() [][]string {
+
+	// open the file
+	csvFile, err := os.Open("../pokedex.csv")
+	if err != nil {
+		fmt.Println("error encountered opening csv file")
```

Comment:
This logging is fine but it's missing the actual error, we should log that as well.

Also it would be better to use the `log` package instead of `fmt.Println` for logging.